orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/ManifestForceCacheRefreshTask.java

```diff
@@ -89,9 +113,21 @@ public TaskResult execute(Stage stage) {
     stageData.manifestNamesByNamespace = manifestNamesByNamespace(stage);
 
     if (refreshManifests(cloudProvider, account, stageData)) {
+      registry.timer(durationTimerId.withTags("success", "true", "outcome", "complete"))
+        .record(duration, TimeUnit.MILLISECONDS);
```

Comment:
I thought this was a different outcome than line 124 below. Especially now that you removed the counter.

Reply:
Line 105 is the different outcome